### ISSUE TITLE
This is for 64-Bit Windows Only?

### ISSUE THREAD

**Issue by semicodin** [Tichau/FileConverter#3]
No instructions for system requirements?  I'm on XP Professional.  Let me know?

**Comment by Tichau** [comment]
Hello, the currently available installer is 64bits only but I can work on a 32 bits version of it. I think it's not very complicated.
But i'm not sure about the windows XP compatibility.

**Comment by Tichau** [comment]
I've worked on a 32 bits version of file converter, it will be available for the next release.
But it will not be compatible with Windows XP, File Converter use the .NET framework 4.5 that is not supported by Windows XP (https://blogs.msdn.microsoft.com/dotnet/p/dotnet45xp/)

**Comment by Tichau** [comment]
I just released a 32 bits version of file converter. Normally it should work on Windows vista, 7, 8 and 10 (tested on windows 7).
It should not work for windows XP as I said in my previous post. Sorry but I do not plan to work on it for now.

Best regards

**Comment by semicodin** [comment]
Tichau I apologize for not getting back to you sooner.  Thank you in any event.  It's funny because a year ago I purchased a Retail Windows 7 that came with both 32bit and 64bit versions of the OS.  I installed Win7 32bit and promtly removed it a few days later because I don't like it.  I don't like change generally, but every procrastinator has to meet his devil sooner or later.

Did you ever find yourself in a situation in which whatever option you choose you're not going to like it, so you might as well throw caution to the wind and act boldly?  Knowing there is a very real possibility that none of my hardware will work with it, I've nonethelsss decided to install the **64bit** version of WIN7 on the next computer I'm building, but I'm also going to _"double down"_ and install **Linux side-by-side**: If I'm going over the cliff I might as well take a couple of operating systems with me dammit! lol It's sort of like using a chainsaw for the first time.  You've just got to cross that threshold and _start cutting._ Thanks again.
